tests/lora/test_lora_layers_cogvideox.py

```diff
@@ -192,3 +192,7 @@ def test_simple_inference_with_text_lora_fused(self):
     @unittest.skip("Text encoder LoRA is not supported in CogVideoX.")
     def test_simple_inference_with_text_lora_save_load(self):
         pass
+
+    @unittest.skip("Not supported in CogVideoX.")
+    def test_simple_inference_with_text_denoiser_multi_adapter_block_lora(self):
+        pass
```

Comment:
Now that we're catching the error appropriately within the code, we should skip this test for unsupported models.